Description: relaxes-too-strict-pep8-requires.patch
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no

--- python-glanceclient-0.7.0.orig/tools/test-requires
+++ python-glanceclient-0.7.0/tools/test-requires
@@ -1,4 +1,4 @@
-distribute>=0.6.24
+distribute>=0.6.23
 
 mox
 nose
@@ -6,6 +6,6 @@ nose-exclude
 nosexcover
 openstack.nose_plugin
 nosehtmloutput
-pep8==1.3.3
+pep8>=1.3.3
 setuptools-git>=0.4
 sphinx>=1.1.2
